mythril/laser/ethereum/gas.py

```diff
@@ -180,10 +180,7 @@ def calculate_sha3_gas(length: int):
     "LOG3": (4 * 375, 4 * 375 + 8 * 32),
     "LOG4": (5 * 375, 5 * 375 + 8 * 32),
     "CREATE": (32000, 32000),
-    "CREATE2": (
-        32000,
-        32000,
-    ),  # TODO: The gas value is dynamic, to be done while implementing create2
+    "CREATE2": (32000, 32000),  # TODO: Make the gas value is dynamic
```

Comment:
I am no English expert but shouldn't it be this?
```suggestion
    "CREATE2": (32000, 32000),  # TODO: Make the gas values dynamic
```